### DISCUSSION TITLE
Illegal Instruction 4 Error

### DISCUSSION THREAD

**Discussion by thejamesheard** [openai/whisper#1299] [Q&A]
I've had Whisper installed through Homebrew on my M1 Mac for a few months and it was running fine until this morning. I upgraded my outdated formula on Homebrew and now I'm getting the following error:

`/usr/local/Cellar/openai-whisper/20230314/libexec/lib/python3.11/site-packages/whisper/timing.py:57: NumbaDeprecationWarning: The 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.
  @numba.jit
Illegal instruction: 4`

Any idea what's causing this?

**Comment by dmartinezh97** [comment]
I am having the same error, also with a macbook air m1.
